include/deal.II/lac/block_vector_base.h

```diff
@@ -2264,6 +2262,17 @@ void BlockVectorBase<VectorType>::extract_subvector_to (ForwardIterator
     }
 }
 
+
+template <typename VectorType>
+inline
+bool BlockVectorBase<VectorType>::supports_distributed_data() const
+{
+  // Create a dummy vector of type VectorType to check
+  // supports_distributed_data even if BlockVectorBase contains no block
+  VectorType dummy;
+  return dummy.supports_distributed_data();
```

Comment:
Right now the variable is ```static``` and totally independent of the state of vector. I don't see how changing the ```static``` variable to a ```static``` function will help with the inheritance problem in the new ```Vector``` interface. The behavior of the function is exactly the same as the behavior of the variable.